_resources/port1.0/group/python-1.0.tcl

```diff
@@ -292,7 +292,9 @@ proc python_get_defaults {var} {
                 # look for "${inc_dir}*" and pick the first one found;
                 # make assumptions if none are found
                 if {[catch {set inc_dirs [glob ${inc_dir}*]}]} {
-                    if {${python.version} < 30} {
+                    # append 'm' suffix if 30 <= PyVer <= 37
+                    # Py27- and Py38+ do not use this suffix
+                    if {${python.version} < 30} || ${python.version} > 37} {
```

Comment:
actually, the ```}``` after ```30``` shouldn't be there... I will push to this PR in a minute and add the revbump for ```py-sip```. If the rebuild of the ```py38-sip``` port now puts the ```sip.h``` in the correct include directory we know this is good to merge.

Reply:
good catch & thx!